faq.md

```diff
@@ -74,14 +74,15 @@ MPTCP extends beyond to various platforms including macOS. But... </summary>
 
 The use of MPTCP on macOS differs from Linux:
 - It is most straightforward when applications use the system's
-  [frameworks](https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/improving_network_reliability_using_multipath_tcp)
+  [frameworks](https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/improving_network_reliability_using_multipath_tcp),
+  which can be done in
+  [C language](https://developer.apple.com/documentation/network) as well.
 - There is some documentation for
   [connectx](https://opensource.apple.com/source/xnu/xnu-7195.81.3/bsd/man/man2/connectx.2.auto.html)
-  and
-  [disconnectx](https://opensource.apple.com/source/xnu/xnu-7195.81.3/bsd/man/man2/disconnectx.2.auto.html)
-  system calls that mention multipath support, however it is not clear what is
-  required to use these interfaces. There does not appear to be public example
-  code.
+  system call that mention multipath support if the socket has been created with
+  the `AF_MULTIPATH` (`39`) domain. However, it is not clear what is required to
+  use these interfaces. There does not appear to be public example code from
+  Apple, so this method doesn't seem to be recommended.
```

Comment:
I've tested this an it works: it correctly establishes an MPTCP connection with a Linux-based server. However, it is unclear what are its limitation, I haven't tested subflow establishment, handover supports, etc. As you mention, it is probably preferable to use the network API from above.

Reply:
Olivier tried with cURL (see https://github.com/matttbe/curl/pull/1) and files from https://check.mptcp.dev/files/: the connection is done with MPTCP, but no additional sublows are created apparently, even with 'sudo'.

Reply:
Thanks for the info